src/flamenco/runtime/fd_executor.c

```diff
@@ -363,6 +363,7 @@ fd_executor_load_transaction_accounts( fd_exec_txn_ctx_t * txn_ctx ) {
   fd_epoch_schedule_t const * schedule = fd_sysvar_cache_epoch_schedule( txn_ctx->slot_ctx->sysvar_cache );
   ulong                       epoch    = fd_slot_to_epoch( schedule, txn_ctx->slot_ctx->slot_bank.slot, NULL );
 
+  uchar accounts_found[txn_ctx->accounts_cnt];
```

Comment:
The usage of this looks a bit iffy to me. The default behavior in Agave is "found is true unless EXPLICITLY set here: https://github.com/anza-xyz/agave/blob/v2.0.9/svm/src/account_loader.rs#L251-L277"

Look at how the default value is set: https://github.com/anza-xyz/agave/blob/v2.0.9/svm/src/account_loader.rs#L311-L314

Your logic seems to be operating on an opposite assumption and I'm not entirely convinced it's correct. This needs a long comment explaining when exactly `accounts_found` is set to true vs false, and the logic should follow after.

Reply:
Maybe your logic is correct, but it's not entirely clear in what cases `account_found` is true or false. Can you clearly document this?